src/main.ts

```diff
@@ -1,20 +1,16 @@
-import '@mdi/font/css/materialdesignicons.min.css'
-import 'vuetify/dist/vuetify.min.css'
```

Comment:
Importing CSS here and on line 17 was causing styles to be duplicated on the page. I'm not sure if we need line 18 or if it can be removed.